pkg/openstack/actuator.go

```diff
@@ -1,12 +1,9 @@
 /*
 Copyright 2019 The OpenShift Authors.
-
```

Comment:
Accidental damage to the copyright notice?

Reply:
Thanks! I have no idea how this could happen...